solidity/contracts/converter/types/liquidity-pool-v1/LiquidityPoolV1Converter.sol

```diff
@@ -697,6 +670,33 @@ contract LiquidityPoolV1Converter is LiquidityPoolConverter {
         return reserveAmounts;
     }
 
+    /**
```

Comment:
Is this just the result of moving a function from one place in the file to another?